docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -35,6 +36,16 @@ IMPORTANT: To customize which indices are checked when you click *Check all*, {s
 
 * *Check a single index*: To check a single index, expand it using the arrow on the left. Checking a single index is faster than checking all indices.
 
+[discrete]
+== Visualize checked indices
+The treemap that appears at the top of the dashboard shows the relative size of your indices. The color of each index's node refers to it's status:
+
+* *Blue:* Not yet checked.
+* *Green:* Checked, no conflicts found.
```

Comment:
Consider replacing _conflicts_  (on this line, and the one below), with _incompatible fields_ if alignment with the UI labels makes sense in this context.